main.js

```diff
@@ -217,15 +225,22 @@ const submitBtn = document.querySelector("#submit-button");
 submitBtn.addEventListener("click", (e) => {
   e.preventDefault();
   if (document.getElementById('city-input').value.length !== 0) {
-    startSearch();
+    // show the loading gif image
+    gifFig.style.display = "flex";
```

Comment:
we should add a line on 228 with canvas.InnerHTML = '' to remove the Spot Check logo while the loading animation displays.